packages/anagram.rb

```diff
@@ -3,52 +3,46 @@
 class Anagram < Package
   description 'finds anagrams or permutations of words in the target phrase'
   homepage 'https://www.fourmilab.ch/anagram/'
-  version '1.5'
+  version '1.5-1'
```

Comment:
No need for version suffix? (Same with other packages: `gegl`, `gimp`, `openjpeg`, `sed`)